docs/understanding-tracking-design/predefined-vs-custom-entities/index.md

```diff
@@ -23,6 +23,10 @@ window.snowplow("newTracker", "sp", "{{COLLECTOR_URL}}", {
 
 ## Custom contexts
 
+:::info
+Please note that for BDP Cloud customers this complexity is abstracted by [the Data Structures Builder](/docs/understanding-tracking-design/managing-data-structures-with-data-structures-builder/index.md).
```

Comment:
What does this mean? What do we expect the users to do with this information? I would rephrase to something more actionable, like:
```suggestion
If you are using BDP Cloud, you can create custom schemas using the [Data Structures Builder](/docs/understanding-tracking-design/managing-data-structures-with-data-structures-builder/index.md) without worrying about how it works under the hood.
```

Reply:
👍